apis/style/main.scss

```diff
@@ -1 +1,18 @@
-// WARN: Hot reloads don't work without this file using cargo leptos v 0.2.0
\ No newline at end of file
+.nested-content {    
```

Comment:
Does this maybe lead to the "normal distributed" indent? 
<img width="324" alt="Screenshot 2024-06-24 at 15 58 56" src="https://github.com/hiveboardgame/hive/assets/3520174/d9f837d1-e5c4-4af3-b3e9-e986876a6f58">
Shouldn't they all just be `1em`?

Reply:
yes im dumb tx